api/src/test/java/org/pmiops/workbench/api/CohortReviewControllerTest.java

```diff
@@ -282,82 +288,75 @@ public void setUp() {
             .cohortReviewId(cohortReview.getCohortReviewId())
             .participantId(2L);
 
-    participantCohortStatus1 =
-        new DbParticipantCohortStatus()
-            .statusEnum(CohortStatus.NOT_REVIEWED)
-            .participantKey(key1)
-            .genderConceptId(TestDemo.MALE.getConceptId())
-            .gender(TestDemo.MALE.getName())
-            .raceConceptId(TestDemo.ASIAN.getConceptId())
-            .race(TestDemo.ASIAN.getName())
-            .ethnicityConceptId(TestDemo.NOT_HISPANIC.getConceptId())
-            .ethnicity(TestDemo.NOT_HISPANIC.getName())
-            .birthDate(new java.sql.Date(today.getTime()))
-            .deceased(false);
-    participantCohortStatus2 =
-        new DbParticipantCohortStatus()
-            .statusEnum(CohortStatus.NOT_REVIEWED)
-            .participantKey(key2)
-            .genderConceptId(TestDemo.FEMALE.getConceptId())
-            .gender(TestDemo.FEMALE.getName())
-            .raceConceptId(TestDemo.WHITE.getConceptId())
-            .race(TestDemo.WHITE.getName())
-            .ethnicityConceptId(TestDemo.NOT_HISPANIC.getConceptId())
-            .ethnicity(TestDemo.NOT_HISPANIC.getName())
-            .birthDate(new java.sql.Date(today.getTime()))
-            .deceased(false);
-
-    participantCohortStatusDao.save(participantCohortStatus1);
-    participantCohortStatusDao.save(participantCohortStatus2);
-
-    stringAnnotationDefinition =
+    participantCohortStatus1 = participantCohortStatusDao.save(new DbParticipantCohortStatus()
```

Comment:
In the previous code, we manually assigned the transient (but member-backed) fields for gender and race, and a couple of others. After changing to only assign the `participantCohortStatusX` members after `save()`, I notice that these guys are still null. I'd expected them to be calculated based on other information in the model if they're truly transient.